### ISSUE TITLE
Inlined loader is blocked by CSP header

### ISSUE THREAD

**Issue by flexchar** [flexchar/youtuberis#2]
The inlined stylesheet and script for the page loader is currently being blocked by the current Content-Security-Policy Header. 

Ref: [Styles](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src), [Scripts](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src)

...

**Comment by flexchar** [comment]
Disqus and VueJS require overriding crucial security such as `'unsafe-inline' 'unsafe-eval'`. Since the website does not contain functionality, that requires interaction with the server-side data storing, this will be postponed.
